Ladybird/Android/src/main/java/org/serenityos/ladybird/WebViewImplementationNative.kt

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2023, Andrew Kaster <akaster@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+package org.serenityos.ladybird
+
+import android.util.Log
+
+/**
+ * Wrapper around WebView::ViewImplementation for use by Kotlin
+ */
+class WebViewImplementationNative {
+
+    constructor() {
```

Comment:
```suggestion
    init {
```

Reply:
init {} complains at me that I need to initialize the `nativeInstance` integer

Reply:
Oh this seems to be because I need to put the `init {}` towards the bottom of the class definition. I feel like there's some convention about member function/member variable declaration order here but that feels like a Later :tm: thing.

Reply:
Yes, the order matters. Can't you just have

```kotlin
class WebViewImplementationNative {
    private val nativeInstance = nativeObjectInit()
    // ...
} 
```
?

Reply:
True. I've refactored it so that we just init it like that. I think using `val` isn't best b/c I want to re-assign the value to 0 after we've disposed of it.